tests/bml_test.c

```diff
@@ -12,7 +12,8 @@ const int NUM_TESTS = 18;
 
 const char *test_name[] = {
     "add", "allocate", "adjacency", "adjungate_triangle", "convert", "copy",
-    "diagonalize", "inverse", "multiply", "norm", "normalize", "scale", "set_diagonal",
+    "diagonalize", "inverse", "multiply", "norm", "normalize", "scale",
+        "set_diagonal",
```

Comment:
Could you move this string into the next line so that it's not by itself? Thanks!